app/src/androidTest/java/com/github/se/cyrcle/ui/profile/ViewProfileScreenTest.kt

```diff
@@ -220,7 +220,7 @@ class ViewProfileScreenTest {
     composeTestRule.onNodeWithTag("EditButton").assertIsDisplayed()
     composeTestRule.onNodeWithTag("DisplayFirstName").assertTextEquals("Jane")
     composeTestRule.onNodeWithTag("DisplayLastName").assertTextEquals("Smith")
-    composeTestRule.onNodeWithTag("DisplayUsername").assertTextEquals("@janesmith")
+    composeTestRule.onNodeWithTag("DisplayUsernameWithLevel").assertTextEquals("[☤92] janesmith")
```

Comment:
These symbols are part of Unicode so there's a good chance that it can display well everywhere. And we're working on an android app anyways for this project so who cares for now.

Reply:
I think the choice of Unicode characters was very thoughtful and very minecraft factions of you

Reply:
Agreed. Good job making sure that this was taken into account, shows you put some good thought into it!!

Reply:
> I think the choice of Unicode characters was very thoughtful and very minecraft factions of you. SPEAKING OF FACTIONS (just kidding)

I literally took minecraft inspiration. I was on a bunch of lifetime leaderboards on hypixel for years ;) haha